docs/amy_database_structure.md

```diff
@@ -85,13 +85,14 @@ The primary tables used in AMY (that will likely appear in every query) are thos
-* `self_organized_workshops_rolled_from_previous` Integer; number of self-organised workshops allowed that was rolled over from previous membership. This should be the same as `self_organized_workshops_rolled_over` in preceding membership
-* `self_organized_workshops_rolled_over` Integer; number of self-organised workshops allowed that was rolled over to successing membership. The same number should be recorded in `self_organized_workshops_rolled_from_previous` in successing membership
-* `seats_instructor_training` Integer; number of seats allowed in instructor training events in the original contract.
-* `additional_instructor_training_seats`  Integer; number of additional seats allowed in instructor training events beyond the original contract.
-* `instructor_training_seats_rolled_from_previous` Integer; number of instructor training seats allowed that was rolled over from previous membership. This should be the same as `instructor_training_seats_rolled_over` in preceding membership
-* `instructor_training_seats_rolled_over` Integer; number of self-organised workshops allowed that was rolled over to successing membership. The same number should be recorded in `instructor_training_seats_rolled_from_previous` in successing membership
+* `public_instructor_training_seats` Integer; number of public seats allowed in instructor training events in the original contract.
+* `additional_public_instructor_training_seats`  Integer; number of additional public seats allowed in instructor training events beyond the original contract.
+* `public_instructor_training_seats_rolled_from_previous` Integer; number of public instructor training seats allowed that was rolled over from previous membership. This should be the same as `public_instructor_training_seats_rolled_over` in preceding membership
+* `public_instructor_training_seats_rolled_over` Integer; number of public instructor training seats allowed that was rolled over to successing membership. The same number should be recorded in `public_instructor_training_seats_rolled_from_previous` in successing membership
+* `inhouse_instructor_training_seats` Integer; number of in-house seats allowed in instructor training events in the original contract.
+* `additional_inhouse_instructor_training_seats`  Integer; number of additional in-house seats allowed in instructor training events beyond the original contract.
+* `inhouse_instructor_training_seats_rolled_from_previous` Integer; number of in-house instructor training seats allowed that was rolled over from previous membership. This should be the same as `inhouse_instructor_training_seats_rolled_over` in preceding membership
+* `inhouse_instructor_training_seats_rolled_over` Integer; number of in-house instructor training seats allowed that was rolled over to successing membership. The same number should be recorded in `inhouse_instructor_training_seats_rolled_from_previous` in successing membership
```

Comment:
By successing do you mean succeeding?

Reply:
I wanted `-ing` form of `successor`, but I'll change to `succeding` since it's "opposite" from preceding. Good catch!